### DISCUSSION TITLE
How to open a second yuv/stream file for side-by-side view?

### DISCUSSION THREAD

**Discussion by behzadrocket** [IENT/YUView#375] [Q&A] [answered]
How to open a second yuv/stream file for side-by-side view?

**Comment by archsr** [comment]
You have to select both the YUV files in left pane

**Comment by ChristianFeldmann** [comment, marked as answer]
You have to select two items in the `playlist` panel (hold down Ctrl). Also you have to activate one of the split view modes. The item you selected first in the playlist will be shown on the left. The item you selected second will be shown on the right.
